.azure-pipelines/client.yml

```diff
@@ -97,6 +97,18 @@ jobs:
           mkdir input && cp dependencies.json input/
         displayName: 'Download dependency checker'
 
+      - task: Maven@3
+        displayName: 'Install azure-sdk-parent'
+        inputs:
+          mavenPomFile: parent/pom.xml
+          options: '$(DefaultOptions) -DskipTests -Dgpg.skip'
+          mavenOptions: '$(LoggingOptions)'
+          javaHomeOption: 'JDKVersion'
+          jdkVersionOption: '1.11'
```

Comment:
We moved to JAVA 11 for track 2?

Reply:
I think it's just the azure pipeline build setting.